Tests/Resources/Fixtures/config/config2.xml

```diff
@@ -11,5 +11,12 @@
 
     <error enable-parent-provider="true" enable-sibling-provider="true" />
 
-    <sitemap default-chan-frequency="never" enabled="true" />
+    <sitemap enabled="true">
+        <configuration default-chan-frequency="never" enabled="true" name="default">
+            <templates>
+                <template format="xml" template="test.xml" />
```

Comment:
Not sure if the nested way is the right one. It was the easiest way of defining a schema and it is the way i expect the values in the `CmfSeoExtension`